jupyterhub_moss/form/option_form.js

```diff
@@ -1,7 +1,215 @@
 const CONFIG_NAME = 'form-config:v1';
+const CUSTOM_ENV_CONFIG_NAME = 'custom-environment-config:v1';
+
+
+function removeAllChildren(node) {
+  while (node.firstChild) {
+    node.removeChild(node.firstChild);
+  }
+}
+
+function createEnvironmentDiv(key, description, path, checked=false) {
+  const div = document.createElement('div');
+  div.setAttribute('class', 'environment-div');
```

Comment:
`div.classList.add()` https://devdocs.io/dom/element/classlist

Reply:
But it may be way easier to declare a `template` element on your page instead. Put the HTML markup in there, give the template an ID, query it from your JS code, and then clone and append its content to the DOM: https://devdocs.io/html/element/template

Reply:
I gave a try to `template`, but it's not reducing much the amount of code (having to set `label`'s `for` and `input`'s `id` doesn't help) and it spread the thing among 2 places so not really worth it in this case IMO.

I use `classList.add` instead.

Reply:
Fair enough ;)